### DISCUSSION TITLE
dragndrop with comfy backend

### DISCUSSION THREAD

**Discussion by ppbrown** [Stability-AI/StableSwarmUI#149] [General]
I just installed it today, with comfyu back end. 
I'm happy to see that drag and drop of previously generated PNG files copies in the workflow.

But.. NEW PNG files I generate through swarm.. dont work for that.
Is this deliberate, or "not implemented yet", or.. ??

I see there are multiple "include medata" flags in the User settings area. I turned them on. including the "generate a text file" one.
BUt the text file does not have the json format expected by comfyui.

I'd really like to see the comfyUI workflow generated by normal operation of stabilityswarmUI. And especialy stil be able to dragndrop that stuff again.

**Comment by mcmonkey4eva** [comment]
That's actually a limitation comfy intends to fix himself rather than me doing it from the swarm end -- Swarm doesn't generate in Comfy "UI Format", it generates in Comfy "API Format", and those are incompatible formats. "UI" can autogenerate "API" but no the reverse -- there are UI-only details (such as the position of the node on the graph as X/Y coordinates for example) that would need to be generated.

It is a planned feature either way to make autogenerated graphs viewable on the noodle editor page.
